tarpc/src/server.rs

```diff
@@ -393,11 +394,7 @@ where
                 Poll::Pending | Poll::Ready(None) => Closed,
             };
 
-            let expiration_status = match self
-                .in_flight_requests_mut()
-                .poll_expired(cx)
-                .map_err(ChannelError::Timer)?
```

Comment:
Can `ChannelError::Timer` variant be removed from the enum now?

Reply:
I mean in the enum definition [here](https://github.com/google/tarpc/blob/8f9ce0d9b698264d6beeb8d1db62947f7818278f/tarpc/src/server.rs#L348). I think `ChannelError::Timer` might no longer be used anywhere.

Reply:
Oh yah, I agree, it should be removed.

Reply:
I tried to remove it but felt it might be needed at some point, what do u think

Reply:
sure, though it can always be added back :)

Reply:
@tikue would like me to still remove still?